pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/source/PulsarSourceConfig.java

```diff
@@ -37,6 +37,7 @@
     private String subscriptionName;
     private Map<String, String> topicSerdeClassNameMap;
     private String typeClassName;
+    private Long timeoutMs;
```

Comment:
Are we ok with one timeout for all values?

Reply:
I think this is ok.  I am not sure there is a use case for different timeouts for different topics.  How long a message is buffered is usually independent of which topic its from.  If a user really wants that to do something like that, they can always create separate functions.